index.html

```diff
@@ -208,7 +208,63 @@ <h2 class="Recent ">About-me</h2>
             </div>
         </div>
 
+        <section class="contact-form">
+            <form action="https://formspree.io/f/mqkoyljj" method="post" >
+                <h1 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+                <ul class="form-ul">
+                    <li class="name-box">
```

Comment:
- [ ] Good job working on the contact form 😃, but you need to fix one part. The link to your image (`icon`) is broken, kindly make sure you fixed it 👍 

![Capture](https://user-images.githubusercontent.com/57408419/230070366-388aba48-b997-4dfe-bddf-3679139eddb5.JPG)